app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -78,6 +75,7 @@ public function execute()
             );
             $this->dataPersistor->clear('contact_us');
         } catch (LocalizedException $e) {
+            $this->logger->error($e);
```

Comment:
I suppose we should not log LocalizedException. LocalizedException intended for showing to user, not for logging.
You should intercept real transport exception and log it. I suggest to do it here
\Magento\Framework\Mail\Transport::sendMessage
\Magento\Email\Model\Transport::sendMessage
 in catch section where actually MailException created

I'd also changed implementation and dont show real transport exception to user as it done 
```
throw new MailException(new Phrase($e->getMessage()), $e);
```
It can be potentially security issue. But create something like:
```
throw new MailException(new Phrase('Mail transport failure'), $e); // don't forget add translation
```

Reply:
Done